gulpfile.js

```diff
@@ -74,9 +72,38 @@ gulp.task('watch', function() {
 	  gulp.watch('app/*', ['deploy-local']);
 });
 
-gulp.task('deploy-local', ['build'], function() {	
-  return gulp.src(['dist/**/*', '!*.zip'])
-    .pipe(gulp.dest(LOCAL_OWA_FOLDER + '\\' + THIS_APP_ID));
+gulp.task('deploy-local', ['build'], function() {
+    var config;
+    var localOwaFolder;
+
+	try {
+	//Will throw exception when there is no config.json file
+	config = require('./config.json');
+    //Will throw exception when data in config.json is corrupted
+	localOwaFolder = config.LOCAL_OWA_FOLDER;
+	}
+	catch (err) {
+	//Create or override config.json file
+	var LOCAL_OWA_FOLDER_DEFAULT = "C:\\\\Users\\\\user\\\\openmrs\\\\conceptdictionary\\\\owa";
+	var JSON_CONFIG_PATTERN_DEFAULT =
+	     "{\n" +
+	     "\"LOCAL_OWA_FOLDER\": " + "\"" + LOCAL_OWA_FOLDER_DEFAULT + "\"" +
+	     "\n}";
+    var stream = fs.createWriteStream("config.json");
+    stream.once('open', function() {
+    stream.write(JSON_CONFIG_PATTERN_DEFAULT);
+    stream.end();
+
+    //Variables fixed
+    config = require('./config.json');
+    localOwaFolder = config.LOCAL_OWA_FOLDER;
+    })
+    }
+    finally {
+    // Result
+    return gulp.src(['dist/**/*', '!*.zip'])
+            .pipe(gulp.dest(localOwaFolder + '/' + THIS_APP_ID));
```

Comment:
This one probably should have '\\' instead of '/'

Originally changed by Rafał this commit (see changes in line 72|79 in gulpfile.js):
https://github.com/rkorytkowski/openmrs-owa-conceptdictionary/commit/6c54fee692bcbbdd9e2b1d1999c86ab08ca35e3b